Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Definitions/DateFormats.cs

```diff
@@ -1,21 +1,27 @@
 ﻿namespace Frends.Excel.ConvertToCSV.Definitions;
 
+/// <summary>
+/// DateFormats.
+/// </summary>
 public enum DateFormats
 {
     /// <summary>
     /// default value
```

Comment:
Please specify the actual formats used. You can actually go to the ConvertToJSON branch / PR and copy from there. I have already done this there and I believe it is the same here.